src/test/java/se/digg/dgc/service/impl/DefaultDGCBarcodeEncoderDecoderTest.java

```diff
@@ -61,6 +61,8 @@ public void testEncodeDecodeBarcode() throws Exception {
     
     final DigitalGreenCertificate dgc2 = decoder.decodeBarcode(barcode.getImage());
     Assert.assertEquals(dgc, dgc2);
+    Assert.assertEquals("MAERTIN", dgc2.getSub().getGnt());
```

Comment:
I think it would be better with a dedicated test case for the MrzParser class.

Reply:
I agree